test/sharing-test.js

```diff
@@ -69,8 +69,8 @@ async function testGetAssetShare() {
       assetId: TEST_SHEET_ID
     });
     
-    if (shares.result && shares.result.length > 0) {
-      const shareId = shares.result[0].id;
+    if (shares.items && shares.items.length > 0) {
```

Comment:
nit but I'd do something like

```suggestion
  if (shares.items?.length > 0) {
```